src/content/docs/dropins/checkout/containers/place-order.mdx

```diff
@@ -0,0 +1,21 @@
+---
+title: PlaceOrder container
+description: Configure the PlaceOrder container to handle the final checkout step, including place order action, button disablement, and main slot management.
+---
+
+import OptionsTable from '@components/OptionsTable.astro';
+
+The `PlaceOrder` container is designed to handle the final step in the checkout process, where the user confirms and places their order. You can configure it to handle the place order action, disable the place order button, and manage the main slot for the place order button.
+
+## PlaceOrder configurations
+
+The `PlaceOrder` container provides the following configuration options:
+
+<OptionsTable
+  compact
+  options={[
+    ['Option', 'Type', 'Req?', 'Description'],
+    ['onPlaceOrder', 'function', 'No', 'Handles the place order action. It takes a PlaceOrderContext object as an argument and returns a Promise<void>.'],
+    ['disabled', 'boolean', 'No', 'Indicates whether the place order button is disabled.']
+  ]}
+/>
```

Comment:
`PlaceOrder` container accepts a function as a prop to be used to perform some validations (like forms, etc):

```handleValidation?: () => boolean;```

**example**:

```
import PlaceOrder from '@dropins/storefront-checkout/containers/PlaceOrder.js';
import { render as CheckoutProvider } from '@dropins/storefront-checkout/render.js';

const $placeOrder = checkoutFragment.querySelector('.checkout__place-order');

CheckoutProvider.render(PlaceOrder, {
  handleValidation: () => {
    let success = true;
    const forms = document.forms;

    const loginForm = forms[LOGIN_FORM_NAME];

    if (loginForm) {
      success = loginForm.checkValidity();
      if (!success) scrollToElement($login);
    }

    const shippingForm = forms[SHIPPING_FORM_NAME];

    if (
      success &&
      shippingFormRef.current &&
      shippingForm &&
      shippingForm.checkVisibility()
    ) {
      success = shippingFormRef.current.handleValidationSubmit(false);
    }

    const billingForm = forms[BILLING_FORM_NAME];

    if (
      success &&
      billingFormRef.current &&
      billingForm &&
      billingForm.checkVisibility()
    ) {
      success = billingFormRef.current.handleValidationSubmit(false);
    }

    return success;
  },
  onPlaceOrder: async () => {
    displayOverlaySpinner();

    try {
      await checkoutApi.placeOrder();
    } catch (error) {
      console.error(error);
      throw error;
    } finally {
      removeOverlaySpinner();
    }
  },
})($placeOrder),
]);
```